pyproject.toml

```diff
@@ -54,7 +56,11 @@ ignore = [
 # preloading of commonly-used modules.
 "zygote.py" = ["E402", "F401"]
 # Ignore uppercase function parameters (e.g. G for a graph)
-"apps/prairielearn/elements/**/*.py" = ["N803"]
+"apps/prairielearn/elements/**/*.py" = [
+    "N803",
+    # Ignore FIXME messages in elements
+    "FIX001"
```

Comment:
Can we use `# noqa` comments instead of globally disabling? Elements are the vast majority of our Python code, it's not clear what value this rule adds if we blanket disable it in most files.

Reply:
Are you fine if I blanket add # noqa messages to elements? My idea was to turn it off for now, and turn it on later.

Reply:
Ok, I don't have as much experience doing those kind of changes so I'll defer to you. Changed in 20efbc639.